src/pages/Post/Post.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable prettier/prettier */

```

Comment:
preco? ked som to zmazal, vyzera, ze tam su ine konce riadkov.. mozno ti treba nieco nastavit, aby to pouzivalo linuxove line-endings? mozno nejaky git setting?
tu sa pise https://docs.github.com/en/github/getting-started-with-github/configuring-git-to-handle-line-endings ze mozno treba pustit `git config --global core.autocrlf true` (ak fungujes na windowse)

Reply:
no, to som spustil, a veľmi to nefungovalo

Reply:
ked chapem spravne, co sa na windowse deje:
- ked naklonujes projekt, prepisu sa konce riadkov na CRLF
- prettier default od v2.0 je opravovat to na LF
- tak.. krici 😄 
vypol som nakoniec riesenie koncov riadkov v `.prettierrc`